projects/cn0506/README.md

```diff
@@ -1,4 +1,4 @@
-# CN0506-RMII HDL Project
+# CN0506 HDL Project
```

Comment:
It will be nice to have some clarification about the environment variable in the Readme file. This could be an example https://github.com/analogdevicesinc/hdl/blob/adaq8092_final/projects/adaq8092_fmc/Readme.md .